### DISCUSSION TITLE
HTML Title Content in eventSources

### DISCUSSION THREAD

**Discussion by mayhew09** [vkurko/calendar#296] [Q&A] [answered]
I would like to ask for a working example of how to supply HTML content for the Event Title through the evenSources URL. The input for eventSources is a string as it is generated from a phpfile and it looks like this `..., "title": "titletext",...` but if I change it to e.g. this `..., "html", "<b>titletext</b>",...` it doesn't work...

**Comment by vkurko** [comment, marked as answer]
It should be  (see [Content](https://github.com/vkurko/calendar?tab=readme-ov-file#content) for more details):

```json
"title": {"html": "<b>titletext</b>"}
```
